crud/select/comparators.lua

```diff
@@ -150,6 +150,15 @@ local function gen_array_cmp_func(target, key_parts)
     end
 end
 
+local cmp_operators_by_tarantool_iter = {
```

Comment:
Please add a comment with description. At first glance I don't understand why it's needed

Reply:
Added a comment to `comparators.get_cmp_operator` function.